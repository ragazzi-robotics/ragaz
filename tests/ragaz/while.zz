def main():

    var i: int = 0
    while True:
        if i == 3:
            break

        var j: float = 0
        while True:
            if j == 3:
                break
            print(j)
            j += 1

        print(i)
        i += 1
