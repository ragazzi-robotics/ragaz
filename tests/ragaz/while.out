0.000000
1.000000
2.000000
0
0.000000
1.000000
2.000000
1
0.000000
1.000000
2.000000
2
